### PR TITLE
Fix capturing logs from child processes

### PR DESCRIPTION
**Pull request by przepompownia**

### CODE REVIEW

File: src/server/parser.ts [hunk 1]
```diff
@@ -35,13 +35,13 @@ function startIndex() {
...
   scanProcess.on('message', (mess) => {
-    const { data, log } = mess
+    const { data, msglog } = mess
     if (data) {
       if (!workspace.isExistsBuffer(data.uri)) {
         workspace.updateBuffer(data.uri, data.node)
       }
     }
-    if (log) {
+    if (mess.log) {
```
Comment (przepompownia) [inline src/server/parser.ts]: As a beginner I have no idea why using `msglog` did not wok there.

**Comment by przepompownia** [comment]
This is only rename to distinguish `log` as a message and an instance of logger.

**Comment by iamcco** [comment]
close this since https://github.com/iamcco/vim-language-server/pull/16 also have the same change.

File: src/server/parser.ts (hunk 1 above)
Comment (iamcco) [reply src/server/parser.ts]: should also change the name which scan emit

**Pull request closed without merging**